website/docs/en/config/entry.mdx

```diff
@@ -145,19 +117,21 @@ module.exports = {
 
 <ApiMeta addedVersion={'0.2.6'} />
 
-- **Type:**
-
-  ```ts
-  type ChunkLoading = false | ChunkLoadingType;
-  type ChunkLoadingType =
-    | ('jsonp' | 'import-scripts' | 'require' | 'async-node' | 'import')
-    | string;
-  ```
+- **Type:** `false | 'jsonp' | 'import-scripts' | 'require' | 'async-node' | 'import' | string`
 
 - **Default:** `undefined`
 
 How this entry load other chunks.
 
+#### EntryDescription.wasmLoading
+
+<ApiMeta addedVersion={'0.2.6'} />
+
+- **Type:** `false | 'fetch' | 'async-node' | string`
+- **Default:** `undefined`
+
+How this entry load WebAssembly modules.
```

Comment:
Is `EntryDescription.wasmLoading` supported?

![image](https://github.com/web-infra-dev/rspack/assets/7237365/f9e77962-6351-4da6-8657-1607842d0120)

Reply:
it is in types of the config, i'm bot sure

Reply:
Maybe the type should be removed util Rspack support it